src/payment/types.rs

```diff
@@ -474,3 +477,475 @@ impl StrictDecode for ShortChannelId {
 impl lightning_encoding::Strategy for ShortChannelId {
     type Strategy = lightning_encoding::strategies::AsStrict;
 }
+
+#[derive(Clone, Debug, From, PartialEq, Eq)]
```

Comment:
`Hash`, `PartialOrd`, `Ord` are missed.
It also should be `Copy`

Reply:
Done